gadgetchains/Snappy/FD/1/chain.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace GadgetChain\Snappy;
+
+class FD1 extends \PHPGGC\GadgetChain\FileDelete
+{
+    public static $version = '1.4.2 <= ?';
```

Comment:
```suggestion
    public static $version = '*';
```

Since your chain is apparently working on all versions.